tsconfig.json

```diff
@@ -15,7 +15,7 @@
     "allowSyntheticDefaultImports": true,
     "forceConsistentCasingInFileNames": true,
     "allowJs": true,
-    "checkJs": true,
+    "checkJs": false,
```

Comment:
I had to disable because it doesn't work with my passport.js setup. And also, I don't get why are we having this check enabled, if we are not using ts it in the backend?